service/src/test/java/greencity/service/ubs/AzureCloudStorageServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package greencity.service.ubs;
+
+import com.azure.storage.blob.*;
+import greencity.constant.ErrorMessage;
+import greencity.exceptions.BadRequestException;
+import greencity.exceptions.image.FileIsNullException;
+import greencity.exceptions.image.FileNotSavedException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.core.env.PropertyResolver;
+import org.springframework.mock.env.MockEnvironment;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class AzureCloudStorageServiceTest {
+
+    @BeforeEach
+    void setUp() {
+        MockEnvironment mockEnvironment = new MockEnvironment();
+        mockEnvironment.setProperty("azure.connection.string", " ");
+        mockEnvironment.setProperty("azure.container.name", " ");
+        propertyResolver = mockEnvironment;
+        azureCloudStorageService = spy(new AzureCloudStorageService(propertyResolver));
+    }
+
+    @Mock
+    private PropertyResolver propertyResolver;
+
+    private AzureCloudStorageService azureCloudStorageService;
+
+    @Mock
+    BlobContainerClient containerClient;
+
+    @Mock
+    BlobClient blobClient;
+
+    @Test
+    void checkUpload() {
+        MultipartFile multipartFile = new MockMultipartFile("Image", "Image".getBytes(StandardCharsets.UTF_8));
+        doReturn(containerClient).when(azureCloudStorageService).containerClient();
+        when(containerClient.getBlobClient(anyString())).thenReturn(blobClient);
+        doReturn("blobUrl").when(blobClient).getBlobUrl();
+        azureCloudStorageService.upload(multipartFile);
+        assertNotNull(multipartFile);
+        assertNotNull(azureCloudStorageService.getConnectionString());
```

Comment:
please check that values are equal to the values you set in `[the setUp method](https://github.com/ita-social-projects/GreenCityUBS/pull/954/files#diff-eaa5ccebcac5b4f17cb3bfd88859fbcc29a6c87fa847746e41789fbeb8d57702R32)`